styles/edit_mode.h

```diff
@@ -0,0 +1,47 @@
+#ifndef STYLES_EDIT_MODE_H
+#define STYLES_EDIT_MODE_H
+
+#include "../functions/int_arg.h"
+#include "../functions/islessthan.h"
+#include "../functions/scale.h"
+#include "../functions/wavlen.h"
+// Edit Mode Style Arguments / Variables
+
+enum RgbArgumentName {
+  BASE_COLOR_ARG = 1, // Primary Base Color
+  ALT_COLOR_ARG = 2, // Alternate or Force Color (free-use)
+  BLAST_COLOR_ARG = 3, // Blast Color
+  CLASH_COLOR_ARG = 4, // Clash Color
+  LOCKUP_COLOR_ARG = 5, // Lockup Color
+  DRAG_COLOR_ARG = 6, // Drag Color
+  LB_COLOR_ARG = 7, // Lightning Block Color
+  STAB_COLOR_ARG = 8, // Stab Color
+  PREON_COLOR_ARG = 9, // PreOn Color
+  IGNITION_COLOR_ARG = 10, // Ignition (Power Up) Color
+  RETRACTION_COLOR_ARG = 11, // Retraction (Cool Down) Color
+  POSTOFF_COLOR_ARG = 12, // PostOff Color
+  SWING_COLOR_ARG = 13, // Swing Color (Responsive Swing Effects)
+  EMITTER_COLOR_ARG = 14, // Emitter Color (Emitter Spark) can be used for PostOff
+  OFF_COLOR_ARG = 15, // Off Color (Color when blade is Off, for crystals and accents)
+};
+
+enum IntArgumentName {
+  STYLE_OPTION_ARG = 16, // Style Option
+  IGNITION_OPTION_ARG = 17, // Ignition Options
```

Comment:
It was done intentionally to keep RgbArg and IntArg grouped together in the .ini since they are also done in separate menus (COLOR and SETTINGS) respectively.  We check for the arg value in styles for these groups independently, if we mixed values then the check in the menu system wouldn't work and users would be changing values that didn't exist in the style.

Reply:
I'm not exactly objecting.
I think it would make more sense to group arguments that belong together together (like arguments related to ignition)
Making the code work is relatively easy.